### ISSUE TITLE
define a primary language

### ISSUE THREAD

**Issue by noamoss** [hasadna/reportit-user#50]
השפה הראשית באתר הזה מוגדרת לאנגלית <html lang="en"> יש להגדיר לעברית 

[3.1.1 Language of Page A](https://www.w3.org/TR/UNDERSTANDING-WCAG20/meaning-doc-lang-id.html)

**Comment by noamoss** [comment]
@akariv please review and approve/update my (generous) TE.

**Comment by noamoss** [comment]
notice the TE should include a multilingual solution --> the main language should be set by the interface lang.
